databricks/koalas/tests/test_dataframe.py

```diff
@@ -186,6 +186,39 @@ def test_dataframe_multiindex_names_level(self):
         self.assert_eq(kdf[("X", "A")].to_pandas().columns.names, pdf[("X", "A")].columns.names)
         self.assert_eq(kdf[("X", "A", "Z")], pdf[("X", "A", "Z")])
 
+    def test_itertuples(self):
+        pdf = pd.DataFrame({"num_legs": [4, 2], "num_wings": [0, 2]}, index=["dog", "hawk"])
```

Comment:
Can we add a test with multi-index column?

Reply:
Sounds good! Added.

It behaved as below in pandas:
```
>>> pdf
origin       CA       WA
info        age children
count color             
1     black   4        0
2     brown   2        2
>>> for row in pdf.itertuples():
...     print(row)
...     
... 
Pandas(Index=(1, 'black'), _1=4, _2=0)
Pandas(Index=(2, 'brown'), _1=2, _2=2)

```